packages/firebase/index.d.ts

```diff
@@ -7859,9 +7859,11 @@ declare namespace firebase.firestore {
     /**
      * Called by the Firestore SDK to convert a custom model object of type T
      * into a plain Javascript object (suitable for writing directly to the
-     * Firestore database).
+     * Firestore database). To use set() with `merge` and `mergeFields`,
```

Comment:
I think the literal `set()` should have backticks.

Reply:
done.